### PR TITLE
FAIR-614 (Responsible person table data is displayed under the wrong …

### PR DESCRIPTION
**Pull request by sjose012**
…table header)

**What changes does this PR bring?**

**Definition of Done**
The following need to be checked off as they are complete as part of the
PR process. If you are unable to mark off an item as complete, then state
in the comments why this is the case.
- [x] Have you built the code locally and confirmed its working?
- [x] Is the build confirmed to be passing on CI?
- [ ] Have you added enough relevant unit tests for your change?
- [ ] Have you added enough relevant E2E tests for your change?
- [ ] Have you updated any relevant documentation as part of this change?
- [ ] Has this change been tested against the Acceptance Criteria?
- [ ] Have you considered how this will be deployed in SIT/Staging/Production?

### CODE REVIEW

File: src/app/resperson/index.njk [hunk 1]
```diff
@@ -56,10 +56,10 @@
                       <button id="edit-edit_reponsiblePerson-{{ loop.index }}" class="govuk-link btn-link" 
                         title="Edit details for {{ resPerson.responsibleGivenName }} {{ resPerson.responsibleSurname }}" 
                         formaction="resperson/edit" formmethod="get"
-                        name="editResponsiblePerson" value="{{resPerson.responsiblePersonId}}">{{resPerson.responsibleGivenName}}
+                        name="editResponsiblePerson" value="{{resPerson.responsiblePersonId}}">{{resPerson.responsibleSurname}}
                       </button>
                     </td>
-                    <td class="govuk-table__cell" scope="row">{{resPerson.responsibleSurname}}</td>
+                    <td class="govuk-table__cell" scope="row">{{resPerson.responsibleGivenName}}</td>
```
Comment (ame-test) [inline src/app/resperson/index.njk:62]: Nit: `scope="row"` should be on the surname element than the resperson given name if it is to be used.

Comment (ame-test) [inline src/app/resperson/index.njk:62]: Nit: `scope="row" is not for `td` elements, is is for `th` elements.
* It is deprecated (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td#scope).

**Review by ame-test — APPROVED** [review]
Approved, nit comment is nit.

File: src/app/resperson/index.njk (hunk 1 above)
Comment (sjose012) [reply src/app/resperson/index.njk:62]: removed the scope from td element

Comment (ame-test) [reply src/app/resperson/index.njk:62]: Thanks.

**Pull request merged**